site/src/components/pages/docs/search/search-list-heading.tsx

```diff
@@ -11,48 +11,49 @@ interface SearchItemHeadingProps {
   closeModal?: () => void;
 }
 
-const SearchListHeading: React.VoidFunctionComponent<
-  SearchItemHeadingProps
-> = ({ searchResult, closeModal }) => {
-  const theme = useTheme();
-  const { title, slug } = searchResult;
+const SearchListHeading = forwardRef<HTMLButtonElement, SearchItemHeadingProps>(
+  ({ searchResult, closeModal }, ref) => {
+    const theme = useTheme();
```

Comment:
I fell into this trap because on some of our custom components that extend MUI components (`<Button />` for example), using `theme` props inside `sx` doesn't work. I've played with it and think I found solution for it, I'll address this in a new PR.

Reply:
Okay cool, can you create an asana task for it and link it in this thread? That way we don't lose track of it

Reply:
https://app.asana.com/0/1201095311341924/1202505277043423/f